packages/spark-core/components/_dividers.scss

```diff
@@ -5,7 +5,6 @@ hr,
 .sprk-c-Divider {
   display: block;
   border-bottom: $sprk-divider-border;
-  border-top: none;
```

Comment:
Hey @corcorankt1 , Why'd we remove the border-top?

Reply:
Hm, let me investigate. There shouldn't be any changes to divider in this pr.